src/screens/SiteDetailScreen.js

```diff
@@ -396,6 +419,9 @@ const styles = StyleSheet.create({
     bottomMargin: {
         marginBottom: 20
     },
+    topMargin: {
+        marginTop: 20
+    },
```

Comment:
GH-98 -- `topMargin` that is used by description, but is also available whenever we need it on this page.